internal/cli/opsmanager/logs/list_key_provider.go

```diff
@@ -0,0 +1,67 @@
+// Copyright 2022 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package logs
+
+import (
+	"github.com/mongodb/mongocli/internal/cli"
+	"github.com/mongodb/mongocli/internal/flag"
+	"github.com/mongodb/mongocli/internal/usage"
+	"github.com/spf13/cobra"
+)
+
+type KeyProviderListOpts struct {
+	cli.GlobalOpts
+	cli.OutputOpts
+	file string
+}
+
+func (opts *KeyProviderListOpts) initStore() func() error {
+	// Keeping for now, not sure if needed
+	return func() error {
+		return nil
+	}
+}
+
+func (opts *KeyProviderListOpts) Run() error {
+	// Run the command. For now printing the file path.
+	return opts.Print(opts.file)
+}
+
+// mongocli om logs listKeyProvider --file <encryptedLogFile>.
```

Comment:
@tibulca FYI will update the spec as well

Reply:
@gssbzn does it mean the scope doc has to be updated too?

Reply:
@tibulca not sure, we don't discuss command structure at the scope level but at specs and I have not seen a spec for this yet

Reply:
this time the two commands were included in the scope in the [user flows](https://docs.google.com/document/d/17zIbN4l1lE-s1r78ivFl-ENKHDBGe70EZ1751IsFETo/edit#bookmark=id.5kg9no6kdyby) section, helping with clarifying what are the required steps and params needed (e.g. localKey vs kmip credentials vs kms credentials).
A draft spec can be foud [here](https://docs.google.com/document/d/1gK_kIe1_q5w4XtzwK8kU7lfQLOaKELUaJnGhKlCu6hs/edit#). Initially we were talking about having the spec only with the details related to the tasks since the scope itself got quite detailed.

Reply:
@tibulca I'll let you discuss with Colm if you need to send an email to LGTMrs of the scope but the current command as described breaks current commands structures and I personally don't nitpick command structures at the scope level as I spec a more detailed discussion at the spec level

Reply:
I don't think we need to send an email here. In the scope there was a suggested flow with examples, but we weren't setting a goal that that is the final command structure.

As Gus mentioned, the final commands are resolved in the spec. You should complete the edits for the spec and share it for team review before going too deep on implementation.